frontend/src/features/admin/users/ManageUsers.test.tsx

```diff
@@ -107,11 +103,6 @@ describe('Manage Users Component', () => {
     expect(table.props.data.length).toBe(2);
   });
 
-  it('displays enabled agencies', () => {
```

Comment:
No longer a `select `option, renders as an `input`. This functionality will be tested by the unit test written for `ParentGroupedFilter.tsx`